nodejs/eks/storageclass.ts

```diff
@@ -20,7 +20,7 @@ import * as pulumi from "@pulumi/pulumi";
 /**
  * EBSVolumeType lists the set of volume types accepted by an EKS storage class.
  */
-export type EBSVolumeType = "io1" | "gp2" | "sc1" | "st1";
+export type EBSVolumeType = "io1" | "gp2" | "gp3" | "sc1" | "st1";
```

Comment:
This is a tricky one. This doesn't refer to the node root volumes, but the storage classes for EKS.

The default in-tree plugin doesn't support gp3.
AWS moved to having a separate EBS CSI driver that users need to install in order to use a gp3 storage class.

I'd keep this separate for now and tackle it if we want to tackle https://github.com/pulumi/pulumi-eks/issues/833

Reply:
Thanks for the extra info! I've dropped the commit for this change.